app/components/public/ticket-list.js

```diff
@@ -1,20 +1,27 @@
 import Ember from 'ember';
 import FormMixin from 'open-event-frontend/mixins/form';
-import { sumBy } from 'lodash';
 
 const { Component, computed } = Ember;
 
 export default Component.extend(FormMixin, {
   classNames: ['ui', 'segments', 'ticket-list'],
 
   hasTicketsInOrder: computed('tickets.@each.orderQuantity', function() {
-    return sumBy(this.get('tickets'), 'orderQuantity') > 0;
```

Comment:
@niranjan94 
I think it is an issue with this.get('tickets') being an object, if I use `this.get('tickets').toArray()` it works nicely. I am changing it to that along with the toArray(), please let me know of a better solution for it, if required